### ISSUE TITLE
Allow to set the retry value

### ISSUE THREAD

**Issue by fabianofa** [hhxsv5/php-sse#5]
Reviwing the code, I struggle to find where I could set a custom timeout between connections. 
I understand that the block below uses a fixed value of 2 seconds:

```
SSE::start():

            ...
            if ($changedData !== false) {
                $event = [
                    'id'    => uniqid('', true),
                    'type'  => $eventType,
                    'data'  => (string)$changedData,
                    'retry' => 2000,//reconnect after 2s
                ];
```

Adding the parameter in the signature would allow the developer to handles how often the server should send data to the client, making changes on the signature and on definition of event like:

```
public function start(Update $update, int $milliRetry, $eventType = null)
{
   ...
                $event = [
                    'id'    => uniqid('', true),
                    'type'  => $eventType,
                    'data'  => (string)$changedData,
                    'retry' => $milliRetry,//reconnect after 2s
                ];
}
```

If approved I could make a Pull Request.

**Comment by hhxsv5** [comment]
Thanks for your feedback. Adding a retry time parameter is perfectly fine, but for compatibility reasons, it will be better to add it at the end and keep the same default value.

**Comment by fabianofa** [comment]
Alright, will do.

**Comment by hhxsv5** [comment]
Merged [v1.1.5](https://github.com/hhxsv5/php-sse/releases/tag/v1.1.5).
